templates/zerver/help/include/desktop-toggle-sidebar-tip.md

```diff
@@ -0,0 +1,4 @@
+!!! tip ""
+
+    In order to show or hide the **leftmost sidebar**, you can click on
+    **View** > **Toggle Sidebar** in the main menu bar or press `Ctrl` + `Shift` + `s`.
```

Comment:
How about something like this?

> To show or hide the **organizations sidebar** on the left, select **Toggle Sidebar** from the **View** menu in the top menu bar.

The keyboard shortcut is shown in the menu, so I don't think we need to explicitly teach it here.